### ISSUE TITLE
[Evaluation Details for X-Codec2 Reconstruction]

### ISSUE THREAD

**Issue by sh-lee-prml** [zhenye234/X-Codec-2.0#29]
Hi @zhenye234 

Thanks for your nice work. I hope to follow your benchmark for codec reconstruction task (Table1)

Could you share the details of evaluation for X-Codec2 Reconstruction (Table 1)

I could reproduce metrics without WER. 

Following your paper, 

I used all samples of Librispeech test-clean subset (2620 samples)

and 

used hubert-large-ls960-ft (https://huggingface.co/facebook/hubert-large-ls960-ft)

However, the WER results are different with Table1.

In my case, the WER of GT is 2.50 while the WER in paper is 1.96. 

Do you use all samples to evaluate the metrics?

or

use some text normalization method before calcurating WER? 
(I just used jiwer without any normalization)

Thanks!

**Comment by zhenye234** [comment]
Yes, I use all samples and use text norm following whisper https://github.dev/openai/whisper/blob/main/notebooks/LibriSpeech.ipynb 
My code is like 
trans_gt =[]
trans_from_audio=[]

for item in tqdm(data):
    filename = wav_dir + item['filename'] +'.wav'
    transcription_gt = item['sentence']
    transcription_gt = normalizer(transcription_gt)
    trans_gt.append(transcription_gt)
    audio, _ = torchaudio.load(filename)
    audio = audio.cuda()

    input_values = processor(audio[0], return_tensors="pt").input_values   
    logits = model(input_values.cuda()).logits
    predicted_ids = torch.argmax(logits, dim=-1)
    transcription = processor.decode(predicted_ids[0])
    transcription = normalizer(transcription)
    trans_from_audio.append(transcription)

wer = jiwer.wer( trans_gt,trans_from_audio    )
print(f"WER: {wer * 100:.2f} %")

**Comment by sh-lee-prml** [comment]
@zhenye234 

Thanks for you prompt reply!

I really appreciate it and thank again for your nice work. I like your philosophy of VQ concatenating the semantic and acoustic features instead of RVQ, and I recently followed this structure for low-bitrate codec modeling :) 

Thanks
